src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -1370,6 +1338,17 @@ R1Tensor const SolverBase::gravityVector() const
   return rval;
 }
 
+bool SolverBase::checkSequentialSolutionIncrements( DomainPartition & GEOS_UNUSED_PARAM( domain ) ) const
+{
+  // default behavior - assume converged
+  return true;
+}
+
+void SolverBase::saveSequentialIterationState( DomainPartition & GEOS_UNUSED_PARAM( domain ) ) const
+{
+  // up to specific solver to save what is needed
```

Comment:
```suggestion
  // up to specific solver to save what is needed
  GEOS_ERROR( "Call to SolverBase::saveSequentialIterationState. Method should be overloaded by the solver" );
```
?

Reply:
then i will end up creating bunch of `saveSequentialIterationState` functions for all solvers used in sequential mode...

Reply:
but isn't that the intended behavior? I mean each sequential solver needs to implement this function, doesn't it?

Reply:
You are right
I just committed your suggested code change